src/arch/armv8/vgicv3.c

```diff
@@ -176,18 +191,12 @@ struct vgic_reg_handler_info irouter_info = {
     vgic_int_set_route_hw,
 };
 
-struct vgic_reg_handler_info vgicr_ctrl_info = {
-    vgicr_emul_ctrl_access,
-    0x4,
-};
-struct vgic_reg_handler_info vgicr_typer_info = {
-    vgicr_emul_typer_access,
-    0xC,
-};
-struct vgic_reg_handler_info vgicr_pidr_info = {
-    vgicr_emul_pidr_access,
-    0x4,
-};
+struct vgic_reg_handler_info vgicr_ctrl_info = { vgicr_emul_ctrl_access, 0x4, 0, 0, 0, NULL, NULL,
```

Comment:
Same here.

Reply:
There were some missing in vgicv2.c too. Both files should be correct in commit 13014b264a97b1a89b9e9a4e75cd2f9865d289d9